src/translations/en.json

```
@@ -1958,7 +1958,14 @@
             },
             "conditional": {
               "name": "Conditional",
-              "description": "The Conditional card displays another card based on entity states."
+              "description": "The Conditional card displays another card based on entity states.",
+              "conditions": "Conditions",
+              "card": "Card",
+              "state_equal": "State is equal to",
+              "state_not_equal": "State is not equal to",
+              "current_state": "current",
```

```suggestion
              "current_state": "Current",
```

should this be capitalized?

---

I'm not sure. It's the "current" in `State (current: 'on')`...